tools/vmd/.shed.yml

```diff
@@ -0,0 +1,18 @@
+name: vmd
+owner: chemteam
+description: "vmd is a package for visualize and analyze trajectories from molecular dynamics (MD) simulations"
+homepage_url: https://github.com/MDAnalysis/mdanalysis
+long_description: |
+    VMD is a powerful tool for analysis of structures and trajectories. Numerous tools for analysis are available under the VMD Main menu item Extensions  $ \rightarrow $ Analysis. In addition to these built-in tools, VMD users often use custom-written scripts to analyze desired properties of the simulated systems. VMD Tcl scripting capabilities are very extensive, and provide boundless opportunities for analysis. In this following section, we will learn how to use built-in VMD features for standard analysis, as well as consider a simple example of scripting.
+remote_repository_url: https://github.com/thatchristoph/vmd-cvs-github/tree/master/vmd
+type: unrestricted
+categories:
+  - Computational chemistry
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "Wrapper for the MDAnalysis package: {{ tool_name }}"
+suite:
+  name: "suite_vmd"
```

Comment:
not sure if you need to duplicate the `description` and `long_description` here? But I never really used the suite parameter do I'm not sure.

Reply:
@chrisbarnettster @bgruening @simonbray Should I remove the long description?

Reply:
I think this is fine.